packages/formation-react/src/components/OMBInfo/OMBInfo.jsx

```diff
@@ -108,12 +116,16 @@ OMBInfo.propTypes = {
   resBurden: PropTypes.number,
 
   /**
-   * OMB control number / form number
+   * OMB control number / form number. If this
+   * omitted, it will not show in the rendered component. Be sure to get
+   * approval from the design council before leaving it out.
```

Comment:
```suggestion
   * OMB control number / form number. If this is omitted,
   * it will not show in the rendered component.
```
- Typo: Missing "is"
- I don't think this has to get design system council approval before use. It should go through the collaboration cycle and if leaving any of this data out is concerning, it should be noted at that time. The design system council is there to make decisions on what goes into the design system, not enforce consistency and correctness for each application team.
  - Feel free to remove this language from the `resBurden` prop type comment as well. I'm not 100% sure why it's there.